scripts/listAIQuotas.sh

```diff
@@ -0,0 +1,153 @@
+                    continue
+                fi
+                
+                # Find the candidate model in the list of models and get the available versions
+                available_versions=()
+                for model in $models; do
+                    local model_name=$(echo $model | cut -f1)
+                    local sku=$(echo $model | cut -f2)
+                    local kind=$(echo $model | cut -f3)
+                    local version=$(echo $model | cut -f4)
+                    if [[  $model_fullname == "$kind.$sku.$model_name" ]]; then
+                        if [[ "$versions" == *"*"* ]] || echo "$versions" | grep -q -w "$version"; then
+                            available_versions+=("$version")
+                        fi
+                    fi
+                done
+                
+                # Skip if no available versions
+                if [ ${#available_versions[@]} -eq 0 ]; then
+                    continue
+                fi
+                
+                available_versions_str=$(printf "; %s" "${available_versions[@]}")
+                available_versions_str=${available_versions_str:1}
+                fetched_quotas_table+=("$location,$model_fullname,$available_versions_str,$(echo "$limit - $current_value" | bc),$limit")
+
+                # skip the rest of the candidate_models
+                break
+            done
+        done
+        unset IFS
+    done
+    
+    if [ ${#fetched_quotas_table[@]} -eq 1 ]; then
+        echo "No quotas found for the candidate models"
+        exit 1
+    fi
+    printf "%s\n" "${fetched_quotas_table[@]}"
+}
+
+check_az() {
+    # Check if Azure CLI is installed. If not, prompt the user to install it
+    if ! command -v az &> /dev/null; then
+        echo "Azure CLI is not installed. Would you like to install it now? (Y/N)"
+        read answer
+        if [ "$answer" == "Y" ]; then
+            curl -L https://aka.ms/InstallAzureCli | bash
+            echo "Azure CLI has been installed. Please login using 'az login' and restart the script."
+            exit 0
+        else
+            echo "Azure CLI is required for this script to run. Exiting."
+            exit 1
+        fi
+    fi
+    
+    subscription=$(az account show --query id -o tsv 2>/dev/null)
+    
+    # Exit if not logged in
+    if [ -z "$subscription" ]; then
+        echo "Please login to your Azure account using 'az login' before running this script"
+        exit 1
+    fi
+    
+    echo "$subscription"
+}
+
+
+# list of locations in which to look for candidate models
+locations=("australiaeast" "eastus" "eastus2" "francecentral" "norwayeast" "swedencentral" "uksouth")
+
+# list of candidate models we need
+candidate_models=(
+    "OpenAI.Standard.gpt-35-turbo:*"
+    "OpenAI.Standard.gpt-4:1106-Preview"
+)
+
+subscription=""
+
+while getopts "m:s:" opt; do
+    case $opt in
+        m)
+            candidate_models=($OPTARG)
+        ;;
+        s )
```

Comment:
Is there an extra space for s)

Reply:
Removed, though it does not affect the functionality.